src/desktop/apps/show/routes.coffee

```diff
@@ -74,44 +76,48 @@ query = """
         display
         city
         state
-        postal_code
+        postal_code: postalCode
         country
         address
-        address_2
-        day_schedules {
-          start_time
-          end_time
-          day_of_week
+        address_2: address2
+        day_schedules: daySchedules {
+          start_time: startTime
+          end_time: endTime
+          day_of_week: dayOfWeek
         }
       }
-      artworks {
-        id
-        _id
-        __id
-        href
-        collecting_institution
-        image {
-          url(version: "large")
-          width
-          height
-          aspect_ratio
-          placeholder
-        }
-        partner {
-          href
-          id
-          type
-          name
-        }
-        artists {
-          public
-          href
-          name
+      artworksConnection(first: 100) {
```

Comment:
@sweir27, Matt and I wanted to get your opinion on this change.

We were previously querying for all artworks and displaying them unpaginated in the show detail pages. Now that the `Show` type no longer has a way to get all artworks unpaginated (as it shouldn't) we will need to implement some kind of pagination to use the artworks connection.

I'm curious to know what the future of these pages are before we start adding pagination to the show details page. In the meantime, we are fetching 100 artworks assuming that this will be enough for most shows until we dedicate time to updating this page.

Reply:
Ah! Yes, the proposed updates to the page involve adding in our standard filter/pagination component. I agree with not re-inventing the wheel here when it's going to be re-built shortly!

100 seems like a good guess-- you may be able to easily verify this assumption with a SQL query to redshift (haven't looked into this myself).

Reply:
Sounds good. I'll make sure that we pick a number that makes sense for the shows we have in production. Just out of curiosity, why do you recommend querying redshift instead of our Gravity database?

Reply:
I did a quick scan of `PartnerShows#artworks_count` on staging, and found that there are about 400 (out of 200,000) shows with more than 100 artworks. The maximum was 663 artworks in a show.

I tried to increase the page size to 1000, but it seems like 100 is the maximum page size that Metaphysics currently allows.

Reply:
@starsirius just pointed out another MP query in the show app that handles pagination on the client-side:

https://github.com/artsy/force/blob/80c6e9f74b4594b24455874a5e224fd5eca1d3bf/src/desktop/apps/show/components/artwork_columns_metaphysics/view.coffee#L22

Maybe I can get away [only querying 25 artworks](https://github.com/artsy/metaphysics/blob/5fab5eb99b505457c7385c8e293c4bf6ec3f677f/src/schema/v1/partner_show.ts#L90-L94) on the server-side and then requesting subsequent pages on the client-side!